app/scripts/controllers/push-platform-notifications/firebase/utils/get-notification-data.ts

```diff
@@ -0,0 +1,94 @@
+// eslint-disable-next-line spaced-comment
+/// <reference lib="WebWorker" />
+
+import { BigNumber } from 'bignumber.js';
+
+type FormatOptions = {
+  decimalPlaces?: number;
+  shouldEllipse?: boolean;
+};
+const defaultFormatOptions = {
+  decimalPlaces: 4,
+};
+
+export function calcTokenAmount(value: string, decimals: string) {
```

Comment:
I see you added a code comment for the same function in shared/lib/transactions-controller-utils.js. Did you intend to import that function here?